.gitignore

```diff
@@ -19,7 +19,6 @@ vendor
 /.vscode/
 /.vscode-server/
 /.vscode-server-insiders/
-.byebug_history
```

Comment:
Moving the location was intentional, as it seemed to fit better next to the `rspec` file and not within these directories.